applications/api.py

```diff
@@ -23,6 +30,8 @@ def get_or_create_bootcamp_application(user, bootcamp_run_id):
         Tuple[BootcampApplication, bool]: The bootcamp application paired with a boolean indicating whether
             or not a new application was created
     """
+    from applications.models import BootcampApplication
```

Comment:
I think moving all the email functions from `applications.api` to `mail.api` would get rid of the recursive import that required placing this here.  On the other hand, the current `mail.api` functions are generic, and these are more specific to applications.

Reply:
Yeah, I think these functions are too specific to applications to be moved to the `mail` app

Reply:
@noisecapella In xpro we created `mail_api.py` modules in several Django apps. That might be a good idea here, esp. since we have 3 mail specific functions in here now. Just generally it seems very un-ideal that we wouldn't be able to import a `models` module from an `api` module

Reply:
Sure, I'll make that change